hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-la.tex

```diff
@@ -1,43 +1,74 @@
-%
-%                  ********** hyph-la.tex *************
-%
-% Copyright 1999-2014 Claudio Beccari
-%                [latin hyphenation patterns]
-%
-% -----------------------------------------------------------------
-% IMPORTANT NOTICE:
-%
-% This program can be redistributed and/or modified under the terms
-% of the LaTeX Project Public License Distributed from CTAN
-% archives in directory macros/latex/base/lppl.txt; either
-% version 1 of the License, or any later version.
-% -----------------------------------------------------------------
-%
+% title: Latin Hyphenation Patterns
+% copyright: Copyright (c) 1999-2016 Claudio Beccari
+%                                    e-mail claudio dot beccari at gmail dot com
```

Comment:
It would be nice to make Claudio's email field consistent across multiple pattern files.